Libraries/Components/ScrollView/ScrollViewViewConfig.js

```diff
@@ -24,6 +24,7 @@ const ScrollViewViewConfig = {
     alwaysBounceHorizontal: true,
     alwaysBounceVertical: true,
     automaticallyAdjustContentInsets: true,
+    automaticallyAdjustKeyboardInsets: false,
```

Comment:
```suggestion
    automaticallyAdjustKeyboardInsets: true,
```
This should help with some of the tests 😉

Reply:
oh, missed that - thanks!